src/plugins/plot/inspector/forms/YAxisForm.vue

```diff
@@ -227,8 +227,8 @@ export default {
             let prefix = 'yAxis';
             if (this.isAdditionalYAxis) {
                 let index = -1;
-                if (this.additionalYAxes) {
-                    index = this.additionalYAxes.findIndex((yAxis) => {
+                if (this.domainObject?.configuration?.additionalYAxes) {
+                    index = this.domainObject?.configuration?.additionalYAxes.findIndex((yAxis) => {
```

Comment:
Actually, that's not true.
We create all the axes in the Plot configuration model so that users can drag into them. But any configuration changes are not saved to the domainObject yet. We should keep this.